components/notifications-service2/pkg/storage/postgres/schema/sql/01_one_dot_zero_migrations.up.sql

```diff
@@ -0,0 +1,65 @@
+-- ONE DOT ZERO MIGRATIONS:
```

Comment:
I'm going to handle migrating the database to a consistent state in a future PR. For now having this file here makes the schema migration handling logic in the plan.sh happy.